### PR TITLE
Add new logging to identify image, pck and idbr file paths

### PR DESCRIPTION
**Pull request by JamesGardiner**
Run `make test` to see the new logging lines. Be aware that output could increase significantly.

### CODE REVIEW

File: transform/transformers/CSTransformer.py [hunk 1]
```diff
@@ -52,12 +52,21 @@ def prepare_archive(self):
         Prepare a list of files to save
         '''
         self.files_to_archive.append((settings.SDX_FTP_DATA_PATH, self.pck_file))
+        self.logger.info("ADDED PCK FILE TO ARCHIVE", file=settings.SDX_FTP_DATA_PATH + self.pck_file)
         self.files_to_archive.append((settings.SDX_FTP_RECEIPT_PATH, self.idbr_file))
```
Comment (eldeal) [inline transform/transformers/CSTransformer.py]: Do we not log out the receipt file like all the other files?

Comment (eldeal) [inline transform/transformers/CSTransformer.py]: Also, sentence case please

File: transform/transformers/ImageTransformer.py [hunk 2]
```diff
@@ -88,6 +88,9 @@ def create_image_index(self):
         template_output = template.render(SDX_FTP_IMAGES_PATH=image_path, images=self.images,
                     response=self.response, creation_time=creation_time)
 
+        msg = "ADDING IMAGE TO INDEX"
```
Comment (eldeal) [inline transform/transformers/ImageTransformer.py]: *ahem* _sentencecase_ :)

Comment (JamesGardiner) [inline transform/transformers/ImageTransformer.py]: Oh dear - where's the bashful emoji when you need it?

**Pull request merged**